scripts/linters/pre_commit_linter.py

```diff
@@ -209,68 +205,72 @@ def _get_data(self, filepath, mode):
         return self._CACHE_DATA_DICT[key]
 
 
-def _get_linters_for_file_extension(file_extension_to_lint):
+def _get_linters_for_file_extension(file_extension_to_lint, name_space, files):
```

Comment:
Here and elsewhere in this file: "namespace" is one word.

Reply:
Done.